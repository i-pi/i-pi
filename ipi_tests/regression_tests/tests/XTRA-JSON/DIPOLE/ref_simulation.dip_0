 #EXTRAS(dipole)# Step:           0 
  5.09712476e+01   6.52735096e+01   1.27065068e+01
 #EXTRAS(dipole)# Step:           1 
  5.06795500e+01   6.48226018e+01   1.28877733e+01
 #EXTRAS(dipole)# Step:           2 
  5.01289464e+01   6.48787246e+01   1.24578050e+01
 #EXTRAS(dipole)# Step:           3 
  4.96519448e+01   6.51314056e+01   1.21280062e+01
 #EXTRAS(dipole)# Step:           4 
  4.94069436e+01   6.56811302e+01   1.15920079e+01
 #EXTRAS(dipole)# Step:           5 
  4.94401466e+01   6.59846590e+01   1.09501650e+01
 #EXTRAS(dipole)# Step:           6 
  4.95352051e+01   6.60147912e+01   1.00690690e+01
 #EXTRAS(dipole)# Step:           7 
  4.96301209e+01   6.59176662e+01   9.22139271e+00
 #EXTRAS(dipole)# Step:           8 
  5.00233425e+01   6.55675804e+01   8.75535740e+00
 #EXTRAS(dipole)# Step:           9 
  5.05745981e+01   6.50100274e+01   8.45841823e+00
 #EXTRAS(dipole)# Step:          10 
  5.13066423e+01   6.46618208e+01   8.40638812e+00
