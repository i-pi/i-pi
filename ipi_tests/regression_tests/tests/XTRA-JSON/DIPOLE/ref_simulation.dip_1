 #EXTRAS(dipole)# Step:           0 
  5.09712476e+01   6.52735096e+01   1.27065068e+01
 #EXTRAS(dipole)# Step:           1 
  5.09788382e+01   6.44783780e+01   1.17204749e+01
 #EXTRAS(dipole)# Step:           2 
  5.06158831e+01   6.40294055e+01   1.04506533e+01
 #EXTRAS(dipole)# Step:           3 
  5.03216148e+01   6.36462290e+01   9.47517846e+00
 #EXTRAS(dipole)# Step:           4 
  4.99692728e+01   6.34203110e+01   8.63894399e+00
 #EXTRAS(dipole)# Step:           5 
  4.92454038e+01   6.29769232e+01   7.77199752e+00
 #EXTRAS(dipole)# Step:           6 
  4.78786257e+01   6.26977879e+01   6.69591889e+00
 #EXTRAS(dipole)# Step:           7 
  4.66451394e+01   6.26698196e+01   5.90667472e+00
 #EXTRAS(dipole)# Step:           8 
  4.58125268e+01   6.30454806e+01   5.30740662e+00
 #EXTRAS(dipole)# Step:           9 
  4.54492881e+01   6.30932864e+01   5.20800530e+00
 #EXTRAS(dipole)# Step:          10 
  4.55244294e+01   6.29911090e+01   5.41162850e+00
