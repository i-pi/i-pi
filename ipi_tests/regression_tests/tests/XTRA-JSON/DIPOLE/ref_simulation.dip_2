 #EXTRAS(dipole)# Step:           0 
  5.09712476e+01   6.52735096e+01   1.27065068e+01
 #EXTRAS(dipole)# Step:           1 
  5.03374134e+01   6.45138828e+01   1.32656211e+01
 #EXTRAS(dipole)# Step:           2 
  4.98074283e+01   6.39408795e+01   1.35575064e+01
 #EXTRAS(dipole)# Step:           3 
  4.96501459e+01   6.34460983e+01   1.36218128e+01
 #EXTRAS(dipole)# Step:           4 
  4.96334949e+01   6.33811895e+01   1.31607440e+01
 #EXTRAS(dipole)# Step:           5 
  4.96984290e+01   6.29290228e+01   1.22440242e+01
 #EXTRAS(dipole)# Step:           6 
  4.94106029e+01   6.29809291e+01   1.08380195e+01
 #EXTRAS(dipole)# Step:           7 
  4.90797107e+01   6.33435242e+01   1.01434230e+01
 #EXTRAS(dipole)# Step:           8 
  4.88472446e+01   6.39979072e+01   9.68661117e+00
 #EXTRAS(dipole)# Step:           9 
  4.83474505e+01   6.48942686e+01   9.24300943e+00
 #EXTRAS(dipole)# Step:          10 
  4.74330099e+01   6.59497870e+01   9.01697055e+00
