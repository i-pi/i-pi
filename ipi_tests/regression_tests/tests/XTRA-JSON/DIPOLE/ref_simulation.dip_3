 #EXTRAS(dipole)# Step:           0 
  5.09712476e+01   6.52735096e+01   1.27065068e+01
 #EXTRAS(dipole)# Step:           1 
  5.14707614e+01   6.48505330e+01   1.24499121e+01
 #EXTRAS(dipole)# Step:           2 
  5.21694039e+01   6.47024673e+01   1.19815985e+01
 #EXTRAS(dipole)# Step:           3 
  5.29742393e+01   6.42766931e+01   1.15722206e+01
 #EXTRAS(dipole)# Step:           4 
  5.34067619e+01   6.39372731e+01   1.13421441e+01
 #EXTRAS(dipole)# Step:           5 
  5.37336383e+01   6.32048341e+01   1.10913461e+01
 #EXTRAS(dipole)# Step:           6 
  5.32611505e+01   6.23514179e+01   1.10100033e+01
 #EXTRAS(dipole)# Step:           7 
  5.28171581e+01   6.16201529e+01   1.14998789e+01
 #EXTRAS(dipole)# Step:           8 
  5.23702199e+01   6.11322020e+01   1.17050465e+01
 #EXTRAS(dipole)# Step:           9 
  5.16823308e+01   6.06324298e+01   1.18642826e+01
 #EXTRAS(dipole)# Step:          10 
  5.07408000e+01   6.02678801e+01   1.18451466e+01
