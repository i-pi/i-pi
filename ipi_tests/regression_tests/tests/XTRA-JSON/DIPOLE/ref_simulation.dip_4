 #EXTRAS(dipole)# Step:           0 
  5.09712476e+01   6.52735096e+01   1.27065068e+01
 #EXTRAS(dipole)# Step:           1 
  5.12108744e+01   6.46381538e+01   1.23124029e+01
 #EXTRAS(dipole)# Step:           2 
  5.14548615e+01   6.43314688e+01   1.22120976e+01
 #EXTRAS(dipole)# Step:           3 
  5.16824649e+01   6.40604098e+01   1.25309848e+01
 #EXTRAS(dipole)# Step:           4 
  5.19151154e+01   6.38200402e+01   1.28220133e+01
 #EXTRAS(dipole)# Step:           5 
  5.21835315e+01   6.32755162e+01   1.31349117e+01
 #EXTRAS(dipole)# Step:           6 
  5.19609057e+01   6.30803565e+01   1.29804627e+01
 #EXTRAS(dipole)# Step:           7 
  5.17309772e+01   6.29599100e+01   1.28687203e+01
 #EXTRAS(dipole)# Step:           8 
  5.15397624e+01   6.25404198e+01   1.26427547e+01
 #EXTRAS(dipole)# Step:           9 
  5.15833079e+01   6.18489344e+01   1.24100652e+01
 #EXTRAS(dipole)# Step:          10 
  5.14822716e+01   6.12557398e+01   1.21727858e+01
