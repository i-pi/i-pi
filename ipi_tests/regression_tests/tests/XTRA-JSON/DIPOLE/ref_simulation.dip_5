 #EXTRAS(dipole)# Step:           0 
  5.09712476e+01   6.52735096e+01   1.27065068e+01
 #EXTRAS(dipole)# Step:           1 
  5.08069528e+01   6.50945047e+01   1.32260040e+01
 #EXTRAS(dipole)# Step:           2 
  5.10365143e+01   6.49459731e+01   1.32665418e+01
 #EXTRAS(dipole)# Step:           3 
  5.14363450e+01   6.47740932e+01   1.34239621e+01
 #EXTRAS(dipole)# Step:           4 
  5.14174496e+01   6.46644273e+01   1.38199801e+01
 #EXTRAS(dipole)# Step:           5 
  5.13266037e+01   6.39956696e+01   1.39914850e+01
 #EXTRAS(dipole)# Step:           6 
  5.08380867e+01   6.34830713e+01   1.36886969e+01
 #EXTRAS(dipole)# Step:           7 
  5.03626264e+01   6.30759538e+01   1.33247233e+01
 #EXTRAS(dipole)# Step:           8 
  4.98023576e+01   6.27480022e+01   1.24725285e+01
 #EXTRAS(dipole)# Step:           9 
  4.92563525e+01   6.20880944e+01   1.14922292e+01
 #EXTRAS(dipole)# Step:          10 
  4.87046138e+01   6.16644445e+01   1.07291219e+01
