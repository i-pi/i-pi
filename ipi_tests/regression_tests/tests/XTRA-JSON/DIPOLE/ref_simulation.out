# column   1     --> step : The current simulation time step.
# column   2     --> time{picosecond} : The elapsed simulation time.
# column   3     --> conserved{electronvolt} : The value of the conserved energy quantity per bead.
# column   4     --> temperature{kelvin} : The current temperature, as obtained from the MD kinetic energy.
# column   5     --> kinetic_cv{electronvolt} : The centroid-virial quantum kinetic energy of the physical system.
# column   6     --> potential{electronvolt} : The physical system potential energy.
# column   7     --> pressure_cv{megapascal} : The quantum estimator for pressure of the physical system.
    0.00000000e+00     0.00000000e+00     1.40739852e+02     2.99432342e+02     2.49606489e+01    -5.99126196e+01    -7.11593389e+02   
    2.00000000e+00     1.00000000e-03     1.41280412e+02     2.81516325e+02     4.20189306e+01    -5.11369663e+01    -5.12055314e+02   
    4.00000000e+00     2.00000000e-03     1.41696373e+02     2.28227701e+02     6.15626407e+01    -4.24295793e+01    -6.28154717e+02   
    6.00000000e+00     3.00000000e-03     1.41511266e+02     2.55897081e+02     6.15551570e+01    -4.66079085e+01    -1.03843965e+03   
    8.00000000e+00     4.00000000e-03     1.41544068e+02     3.18610301e+02     6.12013094e+01    -4.61031771e+01    -6.68402426e+02   
    1.00000000e+01     5.00000000e-03     1.41918021e+02     3.50517435e+02     6.80116041e+01    -3.80943862e+01     2.93669161e+02   
